src/containers/Admin/EditTab/index.tsx

```diff
@@ -101,8 +101,9 @@ const EditTab = (): JSX.Element => {
 
     useEffect(() => {
         let ignoreResponse = false
-
-        const ids = [...newStops, ...nearestStopPlaceIds]
+        const ids = newStops
```

Comment:
For å unngå alle sjekkene om variabelen finnes, så går det ann å defaulte til tom liste:
`const { newStops, newStations, hiddenModes, showMap } = settings || {}`
->
`const { newStops = [], newStations, hiddenModes, showMap } = settings || {}`